### ISSUE TITLE
Talkback and keyboard issues.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#428]
```
When typing in any edit box with a bluetooth keyboard, there appears to be a 
lag between the time you hit the letter and when talkback announces what 
character you typed. And when you try to move through the typed text with your 
keyboard talkback does not announce what character you are on. This has been 
happening for some time now. I am using android 4.4.2, Samsung Galaxy S5Active 
and latest beta of talkback.What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?


What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `AriesThe...@gmail.com` on 7 Feb 2015 at 2:53
